Document/0x05b-Basic-Security_Testing.md

```diff
@@ -138,6 +138,51 @@ config
 
 You'll come across different use cases on how you can use adb commands when testing later in this book. Note that you must define the serialnummer of the target device with the `-s` argument (as shown by the previous code snippet) in case you have multiple devices connected.
 
+##### apktool
+
+apktool is used to unpack Android app packages (APKs). Simply unzipping APKs with the standard `unzip` utility leaves some files unreadable. `AndroidManifest.xml` is encoded into binary XML format which isn’t readable with a text editor. Also, the app resources are still packaged into a single archive file.
```

Comment:
Change first word to Apktool + link. And remove the next paragraph.

Reply:
done